packages/paste-theme-utils/src/utils/colorContrastPairingUtils.ts

```diff
@@ -0,0 +1,205 @@
+import ColorCombos from 'color-combos';
+import type {ColorCombo} from 'color-combos';
+import * as DefaultRawTokenJSON from '@twilio-paste/design-tokens/dist/tokens.raw.json';
+import type {DesignToken, DesignTokensJSON} from '@twilio-paste/design-tokens/types';
+import type {GenericTokenShape, AllGenericTokens} from '@twilio-paste/theme';
```

Comment:
Seeing this import from both design-tokens and theme reinforces wanting to put it in the design-tokens package, personally.